test/e2e/storage/persistent_volumes.go

```diff
@@ -313,6 +320,56 @@ var _ = utils.SIGDescribe("PersistentVolumes", func() {
 				pvc = nil
 			})
 		})
+
+		ginkgo.Context("delete pv prior", func() {
+			ginkgo.AfterEach(func() {
+				framework.Logf("AfterEach: Cleaning up test resources.")
+				if errs := e2epv.PVPVCCleanup(c, ns, pv, pvc); len(errs) > 0 {
+					framework.Failf("AfterEach: Failed to delete PVC and/or PV. Errors: %v", utilerrors.NewAggregate(errs))
+				}
+			})
+
+			// Waiting for the pvc and pvc bounded and then delete the pv prior
+			ginkgo.It("to exercise deletion of PV prior to deletion of PVC for a Bound PV-PVC pair", func() {
+
+				ctx, cancel := context.WithTimeout(context.Background(), csiPodRunningTimeout)
+				defer cancel()
+
+				pv, pvc, err = e2epv.CreatePVPVC(c, f.Timeouts, pvConfig, pvcConfig, ns, false)
```

Comment:
Try NFS as we discussed?
https://github.com/kubernetes/kubernetes/blob/v1.24.0-alpha.4/pkg/apis/core/types.go#L990

Reply:
![image](https://user-images.githubusercontent.com/13997135/160223594-cf3a30f6-280d-47f8-a3ca-6ef2bb62ad54.png)
The default  is nfs.